artifacts/pom.xml

```diff
@@ -139,6 +139,10 @@
 		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
 		<certify-sunbird-plugin.version>0.2.0-SNAPSHOT</certify-sunbird-plugin.version>
 		<certify-sunbird-plugin.fileName>certify-sunbird-plugin.jar</certify-sunbird-plugin.fileName>
+		<!-- certify mosip ida plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mosip-identity-plugin.version>0.3.0-SNAPSHOT</certify-mosip-identity-plugin.version>
```

Comment:
why not 0.2.0-snapshot ? can we not raise this PR directly to the release branch ?

Reply:
changed to 0.2.0-SNAPSHOT. i thought this is  the release branch(release-0.9.0-INJI))